docs/config/reference.mdx

```diff
@@ -815,6 +815,9 @@ Trigger: `+`-separated list of keys and modifiers. Example: `ctrl+a`,
 
   * only a single key input is allowed, `ctrl+a+b` is invalid.
 
+  * for number modifier, write them out in letters instead of the actual
```

Comment:
This is a generated file; would you mind submitting this as a PR upstream to Ghostty?

We're going to make this more clear (#183).

https://github.com/ghostty-org/ghostty/blob/bee21880149129f9373a35e9df0d39c223481709/src/config/Config.zig#L780